src/lib/split-view/split-view-panel/split-view-panel.ts

```diff
@@ -0,0 +1,214 @@
+    'forge-split-view-panel': ISplitViewPanelComponent;
+  }
+
+  interface HTMLElementEventMap {
+    'forge-split-view-panel-resize-start': CustomEvent<null>;
+    'forge-split-view-panel-resize-end': CustomEvent<null>;
+    'forge-split-view-panel-resize': CustomEvent<number>;
+    'forge-split-view-panel-did-open': CustomEvent<null>;
+    'forge-split-view-panel-did-close': CustomEvent<null>;
+  }
+}
+
+@CustomElement({
+  name: SPLIT_VIEW_PANEL_CONSTANTS.elementName,
+  dependencies: [
+    IconComponent,
+    RippleComponent
+  ]
+})
+export class SplitViewPanelComponent extends BaseComponent implements ISplitViewPanelComponent {
+  public static get observedAttributes(): string[] {
+    return [
+      SPLIT_VIEW_PANEL_CONSTANTS.attributes.POSITION,
+      SPLIT_VIEW_PANEL_CONSTANTS.attributes.SIZE,
+      SPLIT_VIEW_PANEL_CONSTANTS.attributes.MIN,
+      SPLIT_VIEW_PANEL_CONSTANTS.attributes.MAX,
+      SPLIT_VIEW_PANEL_CONSTANTS.attributes.LABEL,
+      SPLIT_VIEW_PANEL_CONSTANTS.attributes.OPEN,
+      SPLIT_VIEW_PANEL_CONSTANTS.attributes.DISABLED,
+      SPLIT_VIEW_PANEL_CONSTANTS.attributes.DISABLE_CLOSE,
+      SPLIT_VIEW_PANEL_CONSTANTS.attributes.AUTO_CLOSE
+    ];
+  }
+
+  private _foundation: SplitViewPanelFoundation;
+
+  constructor() {
+    super();
+    IconRegistry.define([tylIconDragVerticalVariant, tylIconDragHandle]);
+    attachShadowTemplate(this, template, styles);
+    this._foundation = new SplitViewPanelFoundation(new SplitViewPanelAdapter(this));
+  }
+
+  public connectedCallback(): void {
+    this._foundation.initialize();
+  }
+
+  public disconnectedCallback(): void {
+    this._foundation.disconnect();
+  }
+
+  public attributeChangedCallback(name: string, oldValue: string, newValue: string): void {
+    switch (name) {
+      case SPLIT_VIEW_PANEL_CONSTANTS.attributes.POSITION:
+        this.position = newValue as SplitViewPanelPosition;
+        break;
+      case SPLIT_VIEW_PANEL_CONSTANTS.attributes.SIZE:
+        this.size = newValue;
+        break;
+      case SPLIT_VIEW_PANEL_CONSTANTS.attributes.MIN:
+        this.min = coerceNumber(newValue);
+        break;
+      case SPLIT_VIEW_PANEL_CONSTANTS.attributes.MAX:
+        if (newValue) {
+          this.max = coerceNumber(newValue);
+        } else {
+          this.max = undefined;
+        }
+        break;
+      case SPLIT_VIEW_PANEL_CONSTANTS.attributes.LABEL:
+        this.label = newValue;
+        break;
+      case SPLIT_VIEW_PANEL_CONSTANTS.attributes.OPEN:
+        this.open = coerceBoolean(newValue);
+        break;
+      case SPLIT_VIEW_PANEL_CONSTANTS.attributes.DISABLED:
+        this.disabled = coerceBoolean(newValue);
+        break;
+      case SPLIT_VIEW_PANEL_CONSTANTS.attributes.DISABLE_CLOSE:
+        this.disableClose = coerceBoolean(newValue);
+        break;
+      case SPLIT_VIEW_PANEL_CONSTANTS.attributes.AUTO_CLOSE:
+        this.autoClose = coerceBoolean(newValue);
+        break;
+    }
+  }
```

Comment:
Setting default attributes on individual panels does not set the underlying property (because the panels always pull their values from the parent `<forge-split-view>` element as the default).

I tested the following attributes specifically, but others may have issues:
- `<forge-split-view-panel auto-close>`
- `<forge-split-view-panel disable>`
- `<forge-split-view-panel disable-close>`
- `<forge-split-view-panel orientation="vertical">` (this is likely an internally used property on an individual panel, correct? We may be able to just not document this at the panel level)

Reply:
The panel's orientation is strictly internal. I didn't expose the property itself through the API, just a `setOrientation()` method out of necessity (the parent needs to access it). The others are the only properties shared between the components so it makes sense that they're affected.

Reply:
This should be fixed now. I changed each of the shared properties to be optional in the panel. If one is set it'll use it, otherwise the parent property will be used.